### DISCUSSION TITLE
Regarding Folder Structure when a new Next project is created

### DISCUSSION THREAD

**Discussion by Mahabal1995** [vercel/next.js#53547] [App Router]
Today I create a Next js app using CLI with the same command that is provided by Next Js official website. But the file and folder structure is totally different from what is present in the tutorials. The project that got created does not contain app.js and index.js, and there is no index.html file also whic causes the actual rendering. I have got two files which are rendering the client side components on the browser - a page.js and a layout.js. but now I need to render the nav bar component in every page I create, where to add that code ? can any one please help.... why these changes in File and folder structrue is there ? if it happens why not the documentation on the Next Js website is not updated ?

**Comment by icyJoseph** [comment]
Hi,

Yeah, you've have chosen the App Router when creating the application using the `CLI`.

If you want to learn App Router, then you need to read up in the docs. The tutorial for App Router is in the making as of now.

If you wan to learn Pages Router (with _app.js, and index.js), re-start the project, with the CLI, and when you are asked:

```console
 Would you like to use App Router? (recommended) › No / Yes
```

Select `No`. And you'll use the Pages router, as it is shown in most tutorials.
